polaris-for-vscode/.vscode/tasks.json

```diff
@@ -0,0 +1,20 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "npm",
+			"script": "watch",
+			"problemMatcher": "$tsc-watch",
+			"isBackground": true,
+			"presentation": {
+				"reveal": "never"
+			},
+			"group": {
+				"kind": "build",
+				"isDefault": true
+			}
+		}
+	]
+}
```

Comment:
I think we can remove this. From my understanding it is running tsc and has nothing to do with the plugin.

Reply:
This file lets the VSCode debugger know what to run when you use the `F5` run task shortcut (the `watch` script). removing this file broke shortcut 😢